webapp/components/CommentForm/CommentForm.vue

```diff
@@ -2,18 +2,18 @@
   <ds-form v-model="form" @submit="handleSubmit">
     <template slot-scope="{ errors }">
       <ds-card>
-        <hc-editor
-          ref="editor"
-          :users="users"
-          :hashtags="null"
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-09-11T15:04:26Z" title="Wednesday, September 11th 2019, 5:04:26 pm +02:00">Sep 11, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -2,18 +2,18 @@
   <ds-form v-model="form" @submit="handleSubmit">
     <template slot-scope="{ errors }">
       <ds-card>
-        <hc-editor
-          ref="editor"
-          :users="users"
-          :hashtags="null"
```


nice, you tested and this is not needed after all, right?

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-09-12T07:20:35Z" title="Thursday, September 12th 2019, 9:20:35 am +02:00">Sep 12, 2019</time>_

---


Yeah !!!